### PR TITLE
Add new hotkey for quick batch deletion on test file (merged to current script)

### PR DESCRIPTION
**Pull request by Gabou26**
Added features from FL studio:

Hold XButton1 to delete notes in piano roll. Mappable to different shortcuts in the settings GUI.

Hold click while holding XButton1 for quick batch deletion. Not necessary but a fun extra feature.

### CODE REVIEW

**Comment by Gabou26** [comment]
There is a bug related to the "Audition Multiples Notes" and "Erase Multiples Notes" being cached/saved only on the GUI settings and not on the base settings. So Even if those options are set as true on the GUI, restarting the .exe won't enable them by default.

**Comment by lokanchung** [comment]
I'd like you to create an issue with reproduction steps. Let's discuss on separate thread.
Regarding the MR, I'll test and and try merge this weekend. I've left a comment so please take a look (it's trivial but I think it's better to be fixed here)

File: sp1.ahk [hunk 1]
```diff
@@ -93,13 +108,19 @@ ButtonOK:
     GuiControlGet, swapZoom,, GuiSwapZoom
     GuiControlGet, auditionNotes,, GuiAuditionNotes
     GuiControlGet, auditionNotesShortcut,, GuiAuditionNotesShortcut
+    GuiControlGet, quickErase,, GuiQuickErase
+    GuiControlGet, quickEraseShortcut,, GuiQuickEraseShortcut
+
+
```
Comment (lokanchung) [inline sp1.ahk]: I'd avoid double newlines without a particular reason.

File: sp1.ahk [hunk 2]
```diff
@@ -156,6 +177,28 @@ AuditionNotesHotkey:
     SendInput {1}
 return
 
+; GabPou's code for quick erase
```
Comment (lokanchung) [inline sp1.ahk]: I think comment isn't needed here. We have git to track contributers :)

Comment (MintyDoggo) [reply sp1.ahk]: Totally forgot to remove that lol

**Comment by lokanchung** [comment]
Done! Thanks for the contribution!

**Comment by lokanchung** [comment]
> There is a bug related to the "Audition Multiples Notes" and "Erase Multiples Notes" being cached/saved only on the GUI settings and not on the base settings. So Even if those options are set as true on the GUI, restarting the .exe won't enable them by default.

I just reproduced it and I found the cause. Should be fixed in the next release :)

**Comment by Gabou26** [comment]
Thanks you rock!

**Pull request merged**